app/build.gradle

```diff
@@ -15,6 +16,7 @@ android {
 
         proguardFile getDefaultProguardFile('proguard-android-optimize.txt')
         proguardFile 'proguard-searchview.pro'
+        consumerProguardFiles 'proguard-appsee.pro'
```

Comment:
I posted this on another PR I believe. You want `proguardFile` and not `consumerProguardFiles`.

`consumerProguardFiles` is for proguard rules that get bundled with a library so the app consuming the library can apply them.